### PR TITLE
Update Xdebug for PHP 7.3 compatibility

### PR DESCRIPTION
**Pull request by comxd**
Xdebug 2.7.0 add compatibility with PHP 7.1+ (7.1, 7.2, 7.3).

### CODE REVIEW

File: distribution/debugging.md [hunk 1]
```diff
@@ -59,9 +59,9 @@ version should be displayed in the output.
...
 
-PHP 7.2.8 (cli) (built: Jul 21 2018 08:09:37) ( NTS )
+PHP 7.3.4 (cli) (built: Apr  6 2019 02:50:26) ( NTS )
 Copyright (c) 1997-2018 The PHP Group
-Zend Engine v3.2.0, Copyright (c) 1998-2018 Zend Technologies
-    with Zend OPcache v7.2.8, Copyright (c) 1999-2018, by Zend Technologies
-    with Xdebug v2.6.0, Copyright (c) 2002-2018, by Derick Rethans
+Zend Engine v3.3.4, Copyright (c) 1998-2018 Zend Technologies
+    with Zend OPcache v7.3.4, Copyright (c) 1999-2018, by Zend Technologies
+    with Xdebug v2.7.1, Copyright (c) 2002-2019, by Derick Rethans
```
Comment (teohhanhui) [inline distribution/debugging.md]: Can we just keep this line instead of the full output?

Comment (comxd) [reply distribution/debugging.md]: You want to remove lines 64-65 and leave only line 66?

Comment (teohhanhui) [reply distribution/debugging.md]: Just `with Xdebug 2.7.1`. Ellipses (…) for the rest.

Comment (comxd) [reply distribution/debugging.md]: Ok I just edited.

**Comment by teohhanhui** [comment]
Squashed and fixed.

**Comment by teohhanhui** [comment]
Thanks @comxd! :tada:

**Pull request merged**